### DISCUSSION TITLE
Does the controller depend on zero-ui running all the time?

### DISCUSSION THREAD

**Discussion by twisteroidambassador** [dec0dOS/zero-ui#70] [Q&A] [answered]
I'm thinking about putting a regular zerotier installation on a low-power server to act as the controller, and running zero-ui on my laptop only when I need to make configuration changes to the controller. Would this work?

**Comment by dec0dOS** [comment, marked as answer]
Hello, @twisteroidambassador. It is possible, but I would not recommend you to do so. ZeroUI is lightweight and would work even on the RPI Zero.
